awesome-books/modules/displayRemove.js

```diff
@@ -0,0 +1,43 @@
+const booksContainer = document.querySelector('#book-list');
+
+// Function displayBooks.
+export default function displayBooks({ title, author, id }) {
+  const newBook = document.createElement('div');
+  newBook.id = id;
+  newBook.className = 'book-wrapper';
+  newBook.innerHTML = `
+      <div class="book-info">
```

Comment:
- [ ] The same thing here, Kindly check my comment above ⬆

> requirement
![image](https://user-images.githubusercontent.com/45231276/207326641-903e6e38-a2f5-47e8-ab45-ebb4693b8559.png)